assets/js/session/index.js

```diff
@@ -461,7 +463,9 @@ function editableElementClicked(event, element) {
  * Focuses a focusable element if the user "tab"s anywhere into it.
  */
 function handleDocumentFocus(hook, event) {
-  const focusableEl = event.target.closest(`[data-focusable-id]`);
+  const focusableEl = event.target.closest
+    ? event.target.closest(`[data-focusable-id]`)
+    : false;
```

Comment:
```suggestion
  const focusableEl = event.target.closest && event.target.closest(`[data-focusable-id]`);
```

Reply:
Here we handle document `focus` event and the target can be `window.document`, which doesn't have `closest`. Interestingly I can reproduce it in Firefox only.